sewer/client.py

```diff
@@ -55,13 +55,14 @@ class Client(object):
     def __init__(
         self,
         domain_name,
-        dns_class,
+        dns_class=None,
```

Comment:
@AlecTroemel do you mind adding a log message in the `__init__` method, maybe somewhere here: https://github.com/komuw/sewer/blob/dbee72514fae1d2763f65c13bc944dfa919406d0/sewer/client.py#L186   
it could be something like: 
```python
self.logger.warning(
                "intialise_warning. parameter `dns_class` is deprecated(it will be removed in the next version of sewer), use `auth_provider` instead."
            )
```

Reply:
good call. I also added throwing a `ValueError` if both dns_client and auth_provider were set, since that doesn't make much sense (although `auth_provider` would win out)